metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/tags/Transformer.java

```diff
@@ -0,0 +1,7 @@
+package com.izettle.metrics.influxdb.tags;
+
+import java.util.Map;
+
+public interface Transformer {
+    Map<String, String> tags(String metricName);
```

Comment:
Wouldn't getTags be a better name (for consistency)? Otherwise, I would suggest changing the getTags methods to tags everywhere.